packages/ui/src/ui/constants/groups.ts

```diff
@@ -0,0 +1,9 @@
+import createActionTypes from '../constants/utils';
```

Comment:
I manually renamed these files via `git mv`, but once again - git decided that it's a new file.